SettingsList.py

```diff
@@ -2481,7 +2484,9 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'startwith': 'Start With',
             'vanilla':   'Vanilla Locations',
             'dungeon':   'Dungeon Only',
-            'keysanity': 'Anywhere'
+            'overworld': 'Overworld Only',
+            'dungeons':  'Any Dungeon',
```

Comment:
I'd suggest 'anydungeon' for the choice key, or 'mydungeon' for the 'Dungeon Only' option, just to keep the choice keys harder to confuse. Same for all the others.

Reply:
I went with `any_dungeon` but done.